pkg/minikube/assets/vm_assets.go

```diff
@@ -66,6 +68,11 @@ func (b *BaseAsset) GetPermissions() string {
 	return b.Permissions
 }
 
+// GetModTime returns mod time
+func (b *BaseAsset) GetModTime() (time.Time, error) {
+	return time.Time{}, errors.New("modtime isn't available for this asset")
```

Comment:
We should find a way to disambiguate between modtime not being supported, and an error resolving the modtime.

What do you think about returning `time.Time{}, nil` here, and adding logic to always copy files where time.IsZero()? That way we don't see so many error messages for dynamically generated configs:

```
I1108 15:15:45.011398  254688 ssh_runner.go:201] error getting modtime for deploy/addons/dashboard/dashboard-clusterrolebinding.yaml: modtime isn't available for this asset
I1108 15:15:45.012617  254688 ssh_runner.go:193] dashboard-clusterrolebinding.yaml: copied 1018 bytes
I1108 15:15:45.018533  254688 ssh_runner.go:155] Checked if deploy/addons/dashboard/dashboard-configmap.yaml exists, but got error: modtime isn't available for this asset
```

Reply:
That makes sense to me!